### ISSUE TITLE
Regression: fourmolu-0.5.0.0 deindents multiline tuples

### ISSUE THREAD

**Issue by brandonchinn178** [fourmolu/fourmolu#148]
Running with `--indentation=2 --comma-style=leading`:

`fourmolu-0.4.0.0`:
```hs
[
  ( "foo"
  , handle onError $ do
      print "asdf"
      doFoo
  )
]
```
`fourmolu-0.5.0.0`:
```hs
[
  ( "foo"
  , handle onError $ do
    print "asdf"
    doFoo
  )
]
```

**Comment by brandonchinn178** [comment]
Seems like this broke when merging in ormolu 0.4.0.0 (https://github.com/fourmolu/fourmolu/pull/138). Unfortunately, we can't really bisect those upstream merges. I'll manually debug

**Comment by brandonchinn178** [comment]
Ok so the issue is the addition of `getEnclosingSpan` in `p_hsExpr'`:
https://github.com/fourmolu/fourmolu/blame/0eb1681adbd4d9cf99f5eadc2381b925bfc251ed/src/Ormolu/Printer/Meat/Declaration/Value.hs#L730

This was added because `ghc-lib-parser-9.2` changed the second argument of `ExplicitTuple` from `Located HsTupArg` to just `HsTupArg`. I believe the fix is to use the new `EpAnn` annotations in the `XPresent` type. While ormolu does not have this exact regression (which only shows up in 2-space indentation), this is probably a good change to make upstream. I'll work on a fix here to get fourmolu 0.5.1 out the door, then i'll cherry-pick it upstream as well
